docs/enhancements/add-external-providers.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Add-Providers-Support
+authors:
+  - "@Gal-Zaidman"
+creation-date: 2021-07-25
+last-updated: 2021-07-25
+---
+
+# Add Providers Support
+
+## Summary
+
+Currently, the assisted installer supports Bare-Metal installations only.
```

Comment:
vSphere is already supported (in 'master' branch at least)

Reply:
well, not this way at least...
I mean the code is in I know that, but we defined criteria for testing, provider detection, documentation, and so on...
I don't have a problem with adding it as supported by the assisted installer if you feel strongly about it...

Reply:
The solution here may be to find more precise language than "supports". AI enables you to install on any platform where you can boot the discovery ISO. But I think what you're trying to get at here is that the resulting cluster's formal "platform" is limited to baremetal or none.

Reply:
- I have no strong feelings against including VSphere in this description so I will add it.
- @mhrivnak added your suggestions as well